.github/scripts/setup-env.sh

```diff
@@ -71,19 +71,18 @@ if [[ $GPU_ARCH_TYPE == 'cuda' ]]; then
 fi
 echo '::endgroup::'
 
-if [[ "${OS_TYPE}" == "windows" ]]; then
-  echo '::group::Install third party dependencies prior to TorchVision install on Windows'
-  # `easy_install`, i.e. `python setup.py` has problems downloading the dependencies due to SSL.
-  # Thus, we install them upfront with `pip` to avoid that.
-  # Instead of fixing the SSL error, we can probably maintain this special case until we switch away from the deprecated
-  # `easy_install` anyway.
-  python setup.py egg_info
-  # The requires.txt cannot be used with `pip install -r` directly. The requirements are listed at the top and the
-  # optional dependencies come in non-standard syntax after a blank line. Thus, we just extract the header.
-  sed -e '/^$/,$d' *.egg-info/requires.txt > requirements.txt
-  pip install --progress-bar=off -r requirements.txt
-  echo '::endgroup::'
-fi
+echo '::group::Install third party dependencies prior to TorchVision install'
+# `easy_install`, i.e. `python setup.py`, has some quirks when installing third-party dependencies. For example:
```

Comment:
specifically do you mean `setup.py install`, `setup.py develop`, something else?

Reply:
Both. Let me clarify